.travis.yml

```
@@ -33,7 +33,7 @@ before_script:
   # https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds
   - flake8 . --count --exit-zero --statistics --select=E,F,W,C90
 script:
-  - coverage run --omit=*/virtualenv/*,setup.py,.eggs/*,*__init__.py,syft/workers/websocket_server.py setup.py test
+  - coverage run --omit=*/virtualenv/*,setup.py,.eggs/*,*__init__.py,syft/workers/websocket_server.py,syft/workers/socketio_* setup.py test
```

Why do we not care about coverage of these files?

---

I've been trying to make the tests not hanging when using `socketio` but with no luck. 
A test like this
```suggestion
hook = sy.TorchHook(torch)
app = Flask("test")
socketio = SocketIO(app)
server_worker = WebsocketIOServerWorker(
    hook, "localhost", 5000, app=app, socketio=socketio, log_msgs=True
)

bob = socketio.test_client(app)

def test_client_id():
    android = socketio.test_client(app)
    android.emit("client_id", "android")
    assert len(server_worker.clients) == 1
    android.disconnect()
```

hangs indefinitely

---

I have a feeling the problem is that starting the app hangs. You can get around this by starting it in a separate process then killing the process when everything is done. This is what we do for websocket workers. The easiest way to figure out exactly what is hanging is to just put print statement every line of the test and see where it gets to

---

I will give it a try. Thanks for the hint